sources/core/Stride.Core.Design/VisualStudio/VisualStudioVersions.cs

```diff
@@ -68,102 +82,117 @@ public IDEInfo(Version version, string displayName, string installationPath, boo
         /// <summary>
         /// <c>true</c> if this IDE has a VSIX installer; otherwise, <c>false</c>.
         /// </summary>
-        public bool HasVsixInstaller => !string.IsNullOrEmpty(VsixInstallerPath) && VsixInstallerVersion != VSIXInstallerVersion.None;
+        public bool HasVsixInstaller => !string.IsNullOrEmpty(VsixInstallerPath);
+
+        /// <summary>
+        /// Thd path to the current user's Visual Studio extension installation directory.
+        /// </summary>
+        public string VsixInstallationPath { get; }
+
+        /// <summary>
+        /// Computes and verifies the location of devenv.exe relative to the given path or null if not found. 
+        /// </summary>
+        /// <param name="idePath">The path of this VS instance</param>
+        /// <returns>The full path to devenv.exe or null.</returns>
+        private string ComputeDevenvPath(string idePath)
```

Comment:
Wouldn't it make more sense as a `bool TryFindDevEnvPath(string, out string)`. Or even just inlined to the call site given that this function is very short and only called once.
Same with `ComputeVsixInstallerPath`.

Reply:
Not sure about the TryFind semantic as the bool wouldn't be used.   I'll go with the suggestion to inline these methods as it makes things more compact without reducing readability.